library/Octane/Utility/ClassPropertyMap.hs

```diff
@@ -86,7 +92,8 @@ getParentClassId className parentCacheId xs =
       xs & filter (\(_, name, _, _) -> name == parentClassName) &
       filter (\(_, _, cacheId, _) -> cacheId == parentCacheId) &
       map (\(classId, _, _, _) -> classId) &
-      Maybe.listToMaybe
+      Maybe.listToMaybe &
+      Maybe.maybe (getParentClassId StrictText.empty parentCacheId xs) Just
```

Comment:
I should probably make `className` a `Maybe` and pass `Nothing` instead of using `empty` as a sentinel value.